espanol/noticias/pregunta-periodista/curar-heridos/curar-heridos.md

```diff
@@ -0,0 +1 @@
+Esta guerra no iba a tardar en reanudarse, los perros de la rep�blica volver�an y hab�a que estar preparados; todos los hombres deb�an estar disponibles para ese momento.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.